newIDE/app/src/ExportAndShare/ShareDialog/index.js

```diff
@@ -347,7 +347,7 @@ const ShareDialog = ({
           onClose={() => setBuildsDialogOpen(false)}
           authenticatedUser={authenticatedUser}
           game={game}
-          onGameUpdated={loadGame}
+          onGameUpdated={setGame}
```

Comment:
I remember changing that `onGameUpdated` a while back so that it would trigger a loadGame() again.
I don't remember why it was needed, but just updating the game received wasn't good enough.
Looks like you've differentiated both cases which seems good to me

Reply:
I've double checked and in the case of SharedDialog, this will delegate this down to `BuildProgressAndActions` which is updating the `publicWebBuildId` of the game.
This seemed to work in my test!